locales/en/email-strings.ftl

```diff
@@ -0,0 +1,79 @@
+
+click-to-verify = 
+  Select the Verify My Email button within 24 hours to confirm your Firefox Monitor account. 
+  Your report will then be on its way.
+verify-my-email = Verify My Email
+
+scan-another-email = Scan another email in {-product-name}.
+
+automated-message = 
+  This is an automated email; if you received it in error, no action is required.
+
+# Without HTML markup reads: We sent this message to [user’s email address] because the email address opted into alerts from Firefox Monitor.
+we-sent-this-alert = We sent this message to { $userEmail } because the email address opted into alerts from {-product-name}.
+
+unsubscribe = unsubscribe
+
+# This string is followed by an Unsubscribe button, and appears in full: If you no longer want Firefox Monitor alerts, unsubscribe.
+unsubscribe-blurb = 
+  If you no longer want {-product-name} alerts,
+
+# A Firefox Monitor Report is an emailed statement from Firefox Monitor containing a list of known data breaches where the user’s email address was found amongst the stolen data.
+firefox-monitor-report = {-product-name} Report 
+report-date = Report Date:
+email-address = Email Address:
+
+# "full report" should be understood to mean the "complete report" or, the complete list of known data breaches that included the user’s information. 
+your-full-report = Here’s your full {-product-name} report, which includes all known data breaches that contain this email address.
+
+report-no-breaches = 
+  Your email address did not appear in our database of known breaches. 
+  But breaches can happen at anytime. Take these steps to keep your personal data safe online.
+
+# A list of next steps someone should take if their information has been involved in a data breach.
+what-to-do-next = What To Do Next
+
+report-headline = 
+  { $breachCount ->
+      [0] So far, so good.
+      [one] Your account appeared in { $breachCount } breach.
+     *[other] Your accounts appeared in { $breachCount } breaches.
+  }
+
+report-subhead =
```

Comment:
This should be split in 2 separate strings, with logic handled in the code, since the number is not in the string and it's needed by most languages.